docs/configuration.rst

```diff
@@ -66,6 +66,12 @@ The following environment variables for the tracer are supported:
      description: |
          Enables sending :ref:`telemetry <Instrumentation Telemetry>` events to the agent.
 
+   DD_TRACE_128_BIT_TRACEID_GENERATION_ENABLED:
+     type: Boolean
+     default: False
+     description: |
+         This configuration enables the generation of 128 bit trace ids to better align with industry standards (ex: `OpenTelemetry <https://opentelemetry.io/docs/reference/specification/trace/api/>`_).
```

Comment:
```suggestion
         This configuration enables the generation of 128 bit trace ids.
```

again, not worth saying _why_ more about what it is. Is it worth mentioning b3/w3c support isn't included yet, or is that out of scope of this setting?